### PR TITLE
Organize App Engine quick fixes (#3345)

### LINKED ISSUE #3345: Organize App Engine quick fixes
See how they show up as four different problems instead of a group:

image

Also, three of them have the same name.

### PR DESCRIPTION
**Pull request by lak-proddev**
Fix for Organize App Engine quick fixes #3345
You may need to change the label of  markers

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.properties [hunk 1]
```diff
@@ -11,3 +11,6 @@ maven.plugin.marker=Obsolete Maven App Engine Plugin
 undefined.servlet.marker=Unresolved Servlet Class
 undefined.servlet.mapping=Undefined Servlet Mapping
 unresolved.jsp.file=Unresolved JSP File
+remove.application.element=Remove application element
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.properties]: This should probably be a description, not a verb. E.g. "Ignored application element"

File: plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.properties [hunk 2]
```diff
@@ -11,3 +11,6 @@ maven.plugin.marker=Obsolete Maven App Engine Plugin
...
+remove.version.element=Remove version element
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.properties]: Ignored version element

File: plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.properties (hunk 1 above)
Comment (lak-proddev) [reply plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.properties]: Agree

File: plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.properties (hunk 2 above)
Comment (lak-proddev) [reply plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.properties]: Yes, agree

**Comment by elharo** [comment]
Thanks!

**Pull request merged**